### PR TITLE
Add community health files

### PR DESCRIPTION
**Pull request by osterman**
## What

This PR introduces several key community health files to our repository:

1. **`SECURITY.md`**: Outlines our security policy and provides guidance on how to report security vulnerabilities.
2. **`SUPPORT.md`**: Details the support avenues available for users and contributors, including guidance on where to find documentation, how to report issues, and where to have discussions.
3. **`CONTRIBUTING.md`**: Provides clear guidelines for those looking to contribute to the repository. This includes instructions for submitting pull requests, reporting bugs, and suggesting enhancements.
4. **`GOVERNANCE.md`**: Describes the governance structure for Cloud Posse's projects, highlighting how decisions are made and the role of community input.

## Why

The inclusion of these documents is vital for enhancing our community's structure and ensuring a transparent, welcoming environment for all contributors and users:

- **`SECURITY.md`**: By establishing a formal security policy, we enhance the security and integrity of our project, ensuring a clear protocol for reporting and handling security issues.
- **`SUPPORT.md`**: This file will guide users and contributors to the right support channels, improving their experience and engagement with our community.
- **`CONTRIBUTING.md`**: Encouraging contributions is essential for the growth of our project. This document provides necessary guidance to new contributors, making the process more accessible and streamlined.
- **`GOVERNANCE.md`**: Establishing a clear governance model is crucial for transparent decision-making. This document emphasizes our commitment to community-driven development and highlights the importance of community input in our processes.

Overall, these documents lay the foundation for a healthy, active, and secure open-source community, aligned with Cloud Posse's commitment to excellence in open source and collaborative development.

## References
- Closes cloudposse/atmos#450

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## Contact
+
+We want to hear from you. Please let us know if there’s something we should add, amend or update as part of our Code of Conduct.
+
+| Contact Method      | Details                                                     |
+|---------------------|-------------------------------------------------------------|
+| **Email**           | [community@cloudposse.com](mailto:community@cloudposse.com) |
+| **Phone**           | [+1 800-358-9018](tel:+18003589018)                         |
+| **Mailing Address** | 440 N Barranca Ave #1430, Covina, CA 91723                  |
```
Comment (nitrocode) [inline CODE_OF_CONDUCT.md:149]: Still in CA ? I thought Cloud Posse was in Texas now

**Review by nitrocode — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Fix problems that are important to users
+- Engage the community in working toward the best possible Cloud Posse
+- Enable a sustainable system for Cloud Posse's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow all instructions in Pull Request template
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that all status checks are passing
+
```
Comment (joe-niland) [inline CONTRIBUTING.md:50]:
```suggestion
4. If you need to request a review on your PR, please do so in the #pr-reviews Slack channel
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- Engage the community in working toward the best possible Cloud Posse
+- Enable a sustainable system for Cloud Posse's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow all instructions in Pull Request template
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that all status checks are passing
+4. If you need to request a review on your PR, please do so in the #pr-reviews Slack channel
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
```
Comment (joe-niland) [inline CONTRIBUTING.md:51]: Fix formatting:
```suggestion

While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (osterman) [reply CODE_OF_CONDUCT.md:149]: Our mailing address is...

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+3.  **Search our Slack for the answer.** We see a lot of the same questions asked over and over again. Do a quick search before asking your question.  
+4.  **Ask Constructive Questions.** When asking your question, mention any research you’ve done, relevant links and GitHub issues, as well as provide any relevant code samples. 
+5.  **Use Markdown Formatting.** When pasting code samples, make sure you use markdown formatting. Code that is not formatted makes it difficult for others to read and reduce the likelihood of getting an answer. For longer blobs of text, make sure you use snippets.**IMPORTANT**: Make sure you have permission to share any code samples because these will be public.
+6.  Use the Appropriate Channel. We have dozens of channels that help us organize all the information and questions asked. The #general channel is only for welcoming new users or asking for help on where to ask a question.
```
Comment (osterman) [inline SUPPORT.md:62]:
```suggestion
6.  **Use the Appropriate Channel.** We have dozens of channels that help us organize all the information and questions asked. The #general channel is only for welcoming new users or asking for help on where to ask a question.
```

**Pull request merged**